objcgen/objcgenerator.cs

```diff
@@ -544,7 +544,11 @@ protected override void Generate (ProcessedProperty property)
 			var property_type = NameGenerator.GetTypeName (pt);
 			if (types.Contains (pt))
 				property_type += " *";
-			headers.WriteLine ($") {property_type} {name};");
+
+			var spacing = " ";
+			if (property_type.EndsWith ("*", StringComparison.Ordinal))
```

Comment:
Since you're using `Ordinal` you can simplify this with
`property_type [property_type.Length - 1] == '*'`

or

```
var separator = property_type [property_type.Length - 1] == '*' ? String.Empty : " ";
```

There is sadly no overload with `char` for `[Starts|Ends]With` :(